make/ibm.mk

```diff
@@ -1,31 +1,54 @@
-VERSION ?= 1.0.0
-
-BUNDLE_IMG ?= ibm-account-iam-operator-bundle:v$(VERSION)
-
-IMG ?= ibm-account-iam-operator:v$(VERSION)
```

Comment:
No need to remove these variables. We're keeping as much of the customizations in ibm.mk as possible.

Refactor changes while keeping this in mind